### PR TITLE
Fix issues on jetbrains support

### PR DESCRIPTION
**Pull request by javiersantossanchez**
fix some small issues on jetbrains support feature

### CODE REVIEW

File: pengwin-setup.d/cpp-vs-clion.sh [hunk 1]
```diff
@@ -51,7 +51,7 @@ function main {
       done
     fi
 
-    bash "${SetupDir}"/jetbrains-support.sh --yes "$@"
+    bash "${SetupDir}"/jetbrains_support.sh --yes "$@"
```
Comment (crramirez) [inline pengwin-setup.d/cpp-vs-clion.sh:54]: The standard in pengwin is to have dashes in the filenames, do please change it here and also the filename

File: pengwin-setup.d/ruby.sh [hunk 2]
```diff
@@ -56,6 +56,8 @@ if (confirm --title "RUBY" --yesno "Would you like to download and install Ruby
   echo "Rehashing rbenv"
   rbenv rehash
 
+  bash "${SetupDir}"/jetbrains_support.sh --yes "$@"
```
Comment (crramirez) [inline pengwin-setup.d/ruby.sh:59]: Change here for dashes

**Review by crramirez — CHANGES_REQUESTED** [review]

File: pengwin-setup.d/cpp-vs-clion.sh (hunk 1 above)
Comment (javiersantossanchez) [reply pengwin-setup.d/cpp-vs-clion.sh]: Done. 
I changed the file name and all the reference

File: pengwin-setup.d/ruby.sh (hunk 2 above)
Comment (javiersantossanchez) [reply pengwin-setup.d/ruby.sh]: Done. 
I changed the file name and all the reference

**Pull request merged**